uiowa/uiowa
# Migration computer science

With the help of Cory and the rest of the development team, here is the migration module for the Computer Science news items. Will solve #3357.

Still to do:

- [x] Add multi-field handling for image gallery field.
- [x] Choose image gallery image format
- [x] Debug inline image replacements
- [ ] New code for inline file updates?

# How to test

- In DevDesktop, clone site and pull database from uiclas application (may not be able to do)
- git checkout migration_computer_science (and the usual stuff)
- blt ds --site=cs.uiowa.edu
- drush @cs.local uli
- Enable "Computer Science Migrate" module in Extend
- Go to Configuration > System > SiteNow Migrate
- Enter default values from comments except for Database name (cs_uiowa_edu) and Base URL (https://cs.uiowa.edu).
- Go to Adminsitration > Structure > Migrations > Edit SiteNow migration group.
- Execute Articles migration (should be at least 257 items).
